moped-etl/prefect/tasks/ecs.py

```diff
@@ -328,6 +330,9 @@ def create_task_definition(basename, database):
                         "value": HASURA_GRAPHQL_DATABASE_URL,
                     },
                     {"name": "HASURA_ADMIN_SECRET", "value": HASURA_ADMIN_SECRET},
+                    #  This depends on the Moped API endpoint returned from API commission tasks, add /events/ to end
+                    # {"name": "MOPED_API_EVENTS_URL", "value": MOPED_API_EVENTS_URL},
+                    {"name": "MOPED_API_APIKEY", "value": MOPED_API_APIKEY},
```

Comment:
We can tie this up when we start stitching things together - it is getting close!